### PR TITLE
Update docs links to code.enthought.com

### PR DESCRIPTION
**Pull request by kitchoi**
For #256 

@jdmarch can you take a look please?

### CODE REVIEW

File: docs/source/mayavi/auto/adjust_cropping_extents.py [hunk 1]
```diff
@@ -14,9 +14,9 @@
 are modified to propagate them to the filter. For more information
 on creating GUIs with Traits:
 
-    http://code.enthought.com/projects/traits/docs/html/tutorials/traits_ui_scientific_app.html
+    http://docs.enthought.com/traitsui/tutorials/traits_ui_scientific_app.html
 
-    http://code.enthought.com/projects/traits/documentation.php
+    http://docs.enthought.com/traits
```
Comment (jdmarch) [inline docs/source/mayavi/auto/adjust_cropping_extents.py]: Let's also include 
https://support.enthought.com/hc/en-us/articles/204469620-Introductory-materials-for-Traits-and-Traits-UI

File: docs/source/mayavi/building_applications.rst [hunk 2]
```diff
@@ -29,9 +29,9 @@ callbacks when they are modified. In particular this means that when a
 visualization object is modified, the scene can update automatically.
 
 We strongly suggest that you refer to the `Traits` 
-`documentation <http://code.enthought.com/projects/traits.documentation.php>`_
+`documentation <http://docs.enthought.com/traits>`_
 for more details, and to the 
-`tutorial <http://code.enthought.com/projects/traits/docs/html/tutorials/traits_ui_scientific_app.html>`_
+`tutorial <http://docs.enthought.com/traitsui/tutorials/traits_ui_scientific_app.html>`_
 for a quick introduction.
```
Comment (jdmarch) [inline docs/source/mayavi/building_applications.rst]: and also here

**Comment by jdmarch** [comment]
@kitchoi LGTM. Suggest adding one more link, see inline notes.

**Comment by kitchoi** [comment]
@jdmarch Thanks for reviewing and the link!  Since the "Writing a graphical application for scientific programming using TraitsUI" page is already linked from the support page, I have replaced the former with the latter.

**Comment by jdmarch** [comment]
+1

**Comment by kitchoi** [comment]
Thanks.  Merging.

**Pull request merged**